portfolio/src/main/java/com/google/sps/servlets/MarkerServlet.java

```diff
@@ -99,4 +125,79 @@ public void storeMarker(Marker marker) {
 
     datastore.put(markerEntity);
   }
+
+  public boolean repeatMarkers(double reportLat, double reportLng, String reportDate,
```

Comment:
Why is this public?
same for the new methods below?

Reply:
Already change them to private